src/openforms/urls.py

```diff
@@ -36,6 +36,10 @@
         name="admin_email_test",
     ),
     path("admin/hijack/", include("hijack.urls")),
+    path(
+        "admin/config/",
+        decorator_include(login_required, "openforms.config.urls", namespace="config"),
```

Comment:
that should be at least `staff_required` instead of `login_required`, and please add some access-control tests for this view :)